pkg/machines/libvirt.es6

```diff
@@ -218,12 +223,79 @@ LIBVIRT_PROVIDER = {
                 dispatch(checkLibvirtStatus(libvirtServiceName));
                 startEventMonitor(dispatch, connectionName, libvirtServiceName);
                 doGetAllVms(dispatch, connectionName);
+                dispatch(getStoragePools(connectionName));
             };
         }
 
         return unknownConnectionName(getAllVms, libvirtServiceName);
     },
 
+    /**
+     * Retrieves list of libvirt "storage pools" for particular connection.
+     */
+    GET_STORAGE_POOLS({ connectionName }) {
+        logDebug(`${this.name}.GET_STORAGE_POOLS(connectionName='${connectionName}')`);
+
+        const connection = VMS_CONFIG.Virsh.connections[connectionName].params.join(' ');
+        // Workaround: virsh v1.3.1 in ubuntu-1604 does not support '--name' parameter
+        const command = `virsh ${connection} -r pool-list --type dir | grep active | awk '{print $1 }'`;
+        let poolList = '';
+        // TODO: add support for other pool types then just the "directory"
+        return dispatch => cockpit
+                .script(command, null, { err: "message", environ: ['LC_ALL=en_US.UTF-8'] })
```

Comment:
Should be okay, but in general `C.UTF-8` is better as that's always available, unlike en_US.UTF-8 - if that doesn't exist, you end up with C and possibly get breakage with non-ASCII chars.

Reply:
Good catch, it's a generic issue for the rest of `libvirt.es6`. I'll handle that in a follow-up consistently. Is it ok, please?

Reply:
Actually I just noticed a week or so ago that C.UTF-8 is not available yet on RHEL-7, argh. So we still don't have any UTF-8 locale which is guaranteed to exist everywhere. I recently committed a really [ugly workaround](https://github.com/cockpit-project/cockpit/commit/db30e3c0d0ffb5fcf) to the realm module that tries both. But my gut feeling is that en_US.UTF-8 is indeed a bit safer for now, particularly for RHEL. Sorry for the bad advice!

Reply:
np, thanks for digging this information out.
I will revert to en_US.UTF-8